terraform/aks/config/production.tfvars.json

```diff
@@ -0,0 +1,18 @@
+{
+    "cluster": "production",
+    "namespace": "git-production",
+    "environment": "production",
+    "azure_enable_backup_storage": true,
+    "enable_monitoring": true,
+    "infra_key_vault_name": "s189p01-gse-pd-inf-kv",
+    "statuscake_password_name": "SC-PASSWORD",
+    "sidekiq_replicas" : 2,
+    "sidekiq_memory_max" : "2Gi",
+    "key_vault_resource_group":  "s189p01-gse-pd-rg",
+    "statuscake_alerts": {
+      "alert": {
+        "website_url": [ "https://get-school-experience-production.teacherservices.cloud/healthcheck.json" ],
+        "contact_group": [282783]
+      }
+    }
+}
```

Comment:
at some point you may need to add prod postgres settings for db size, ha and maint window e.g.
  "postgres_flexible_server_sku": "GP_Standard_D4ds_v5",
  "postgres_enable_high_availability": true,
  "postgres_azure_maintenance_window": {
    "day_of_week": 0,
    "start_hour": 3,
    "start_minute": 0
  }

Reply:
done